### ISSUE TITLE
Error returned - Page is in a non-English language

### ISSUE THREAD

**Issue by jmcontreras1213** [worktile/ngx-gantt#136]
Hi,

I'm getting this error message:

npm ERR! 404 Not Found - GET https://registry.npmjs.org/@worktile%2fngx-gantt - Not found
npm ERR! 404 
npm ERR! 404  '@worktile/ngx-gantt@*' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

**Comment by HandsomeButterball** [comment]
Sorry, we wrote the wrong way in the README file, now it has been modified, you can try again. The correct way to use it is 
`$ npm i @worktile/gantt --save`

**Comment by jmcontreras1213** [comment]
Thanks - I changed it now I received this error message:

% npm i @worktile/gantt --save
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: c4-e-portal-repository@0.0.0
npm ERR! Found: @angular/common@11.2.14
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"~11.2.10" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@"^12.0.0" from @worktile/gantt@12.0.0
npm ERR! node_modules/@worktile/gantt
npm ERR!   @worktile/gantt@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
